packages/rocketchat-ui-account/client/accountFlex.html

```diff
@@ -1,13 +1,15 @@
 <template name="accountFlex">
 	<aside class="sidebar-light" role="navigation">
-		<header class="sidebar__header">
-			{{> accountBox lighten=true modifier="--account" }}
-			<button class="sidebar-flex__back-button" data-action="back">
-				{{> icon block="sidebar-flex__back-icon" icon="back"}}
-				<span>{{_ "Back_to_chat"}}</span>
-			</button>
-		</header>
-		<div class="rooms-list">
+		{{#unless embeddedVersion}}
+			<header class="sidebar__header">
+				{{> accountBox lighten=true modifier="--account" }}
```

Comment:
Remove this account box

Reply:
I can remove that, but I won't do that on this PR. This PR is about embedded view and not about accountBox.